notebooks/260-pix2struct-docvqa/260-pix2struct-docvqa.ipynb

```diff
@@ -0,0 +1,381 @@
+   "id": "47b234a5-5d39-4fa3-a528-46a8041b0a01",
+   "metadata": {},
+   "source": [
+    "## Select inference device [$\\Uparrow$](#Table-of-content:)\n",
+    "\n",
+    "select device from dropdown list for running inference using OpenVINO"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "424083a9-2883-4285-9ccb-47f5d3fad407",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "application/vnd.jupyter.widget-view+json": {
+       "model_id": "ba0ee376dbb544b2b84b5e07292f03d8",
+       "version_major": 2,
+       "version_minor": 0
+      },
+      "text/plain": [
+       "Dropdown(description='Device:', index=2, options=('CPU', 'GPU', 'AUTO'), value='AUTO')"
+      ]
+     },
+     "execution_count": 3,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "import ipywidgets as widgets\n",
+    "import openvino as ov\n",
+    "\n",
+    "core = ov.Core()\n",
+    "\n",
+    "device = widgets.Dropdown(\n",
+    "    options=core.available_devices + [\"AUTO\"],\n",
+    "    value='AUTO',\n",
+    "    description='Device:',\n",
+    "    disabled=False,\n",
+    ")\n",
+    "\n",
+    "device"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "65363dcf-c7c2-494c-aa68-4650198264e0",
+   "metadata": {},
+   "source": [
+    "## Test model inference [$\\Uparrow$](#Table-of-content:)\n",
+    "\n",
+    "The diagram bellow demonstrates how model works:\n",
+    "![pix2struct_diagram.png](https://github.com/openvinotoolkit/openvino_notebooks/assets/29454499/c7456b17-0687-4aa9-851b-267bff3dac79)\n",
+    "\n",
+    "For running model inference we should preprocess data first. `Pix2StructProcessor` is responsible for preparing input data and decoding output for original PyTorch model and easily can be reused for running with Optimum Intel model. Then `OVModelForPix2Struct.generate` method will launch answer generation. Finally, generated answer token indices should be decoded in text format by `Pix2StructProcessor.decode`"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "3ef6bd20-dab0-4237-b77f-34eb21262d4f",
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "Compiling the encoder to AUTO ...\n",
+      "Compiling the decoder to AUTO ...\n",
+      "Compiling the decoder to AUTO ...\n"
+     ]
+    }
+   ],
+   "source": [
+    "from transformers import Pix2StructProcessor\n",
+    "\n",
+    "processor = Pix2StructProcessor.from_pretrained(model_id)\n",
+    "ov_model = OVModelForPix2Struct.from_pretrained(model_dir, device=device.value)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "83a2ac29-80d9-4df0-b4e5-584a735f09e7",
+   "metadata": {},
+   "source": [
+    "Let's see model in action. For testing model, we will use screenshot from [OpenVINO documentation](https://docs.openvino.ai/2023.1/get_started.html#openvino-advanced-features)"
```

Comment:
Looks like it should be `the model`. And in other cases too.

Reply:
articles added and verified with grammarly